.prettierrc.js

```diff
@@ -0,0 +1,15 @@
+var commonDxTeamPrettierConfig = require('@fingerprintjs/prettier-config-dx-team')
+
+/** @type {import('prettier').Options} */
+module.exports = {
+  ...commonDxTeamPrettierConfig,
+  overrides: [
+    {
+      files: '*.md',
+      options: {
+        printWidth: 80,
+        singleQuote: true,
+      },
+    },
+  ],
```

Comment:
Question - would it make sense to include this in the shared prettier configuration?

Reply:
I think so! I can make a PR there as well.

Reply:
No idea if I did this right, review appreciated: https://github.com/fingerprintjs/dx-team-toolkit/pull/60

Reply:
Yep, looks good. Thanks!

Once the PR is merged, let's revert this change here 🙂

Reply:
@TheUnderScorer I upgraded the dependency and reverted the change, thank you!

Reply:
Any idea why the `lint` job is suddenly failing? It works fine for me locally. I tried to fix it with a flag but failed

Reply:
It could be a issue with a lockfile. If it passes now, then probably all is good. Thanks @JuroUhlar!